.github/workflows/python-testing-casatools.yml

```diff
@@ -0,0 +1,20 @@
+name: "Python Testing with casatools"
+
+on:
+  push:
+    paths-ignore:
+      - "**.md"
+      - "**.rst"
+  pull_request:
+    paths-ignore:
+      - "**.md"
+      - "**.rst"
+
+jobs:
+  call-testing-with-casatools:
+    uses: casangi/toolviper/.github/workflows/python-testing-casatools-template.yml@main
```

Comment:
The current testing goes over the Py3.11|Py3.12 + Linux |macOS matrix, which is tied to the joint constraints in terms of the Python compatibility of xradio (i.e., from 3.11 to 3.13) and the availability of casatools 6.7.0 wheels:
	https://pypi.org/project/casatools/6.7.0.31/#files
Since NRAO is not releasing casa6 monolithic distributions in Python 3.11, we probably could drop the Py3.11 test from the template to save some GH action minutes.